src/main/kotlin/uk/gov/justice/digital/hmpps/hmppstier/service/TierCalculationService.kt

```diff
@@ -67,10 +58,125 @@ class TierCalculationService(
     } else {
       log.info("Found latest tier calculation for $crn")
     }
-
     return calculation
   }
 
```

Comment:
Just moved the code from a component into the service as used to pass 'OldCalcuation' and 'NewCalculation' in, but now we just have one let of BL it doesn't make sense to have the extra complexity.